blocks/blocks.js

```diff
@@ -196,6 +195,20 @@ const createValidators = () => {
     }
   }
 
+  // Create a color validator
+  // This can either be a column or " "
+  const validateColor = (columnName, pattern) => {
+    return function () {
+      const field = this.getField(columnName)
+      field.setValidator((newValue) => {
+        if (newValue.match(pattern) || " ") {
```

Comment:
This expression is always true: if `newValue.match(pattern)` fails, it then or's with a non-empty string, which is truthy.